`make install` fails on Ubuntu

```
***** src/timers/ *****
***** src/timers/generic/ *****
***** src/qio/regexp/none/ *****
***** src/qio/auxFilesys/hdfs/ *****
***** src/qio/auxFilesys/curl/ *****
***** src/qio/ *****
***** src/ *****
***** ./ *****
cd modules && make
make[2]: Nothing to be done for 'default'.
Exiting: Installation path not configured.
         See ./configure --help
Makefile:190: recipe for target 'install' failed
make: *** [install] Error 255
[Info] Running minimal test script: $CHPL_HOME/util/test/checkChplInstall
[Info] Found executable chpl in /usr/local/bin/chpl.
[Info] Found $CHPL_HOME directory: /home/buddha314/chapel
[Info] /home/buddha314/.chpl does not exist. Creating it.
[Info] Temporary test job directory: /home/buddha314/.chpl/chapel-test-K0SBY
[Info] Compiling $CHPL_HOME/test/release/examples/hello6-taskpar-dist.chpl
[Fail] Test job failed to compile - Chapel is not installed correctly
[Fail] Compilation output:
$CHPL_HOME/modules/internal/ChapelBase.chpl:102: error: unknown pragma: "last resort"
$CHPL_HOME/modules/internal/ChapelBase.chpl:929: error: unknown pragma: "down end count fn"
$CHPL_HOME/modules/internal/ChapelBase.chpl:941: error: unknown pragma: "unchecked throws"
$CHPL_HOME/modules/internal/ChapelBase.chpl:970: error: unknown pragma: "unchecked throws"
$CHPL_HOME/modules/internal/ChapelBase.chpl:989: internal error: BUI0697 chpl Version 1.16.0 pre-release (-999)

Internal errors indicate a bug in the Chapel compiler ("It's us, not you"),
and we're sorry for the hassle.  We would appreciate your reporting this bug --
please see http://chapel.cray.com/bugs.html for instructions.  In the meantime,
the filename + line number above may be useful in working around the issue.
Makefile:184: recipe for target 'check' failed
make: *** [check] Error 1
```

Perhaps this is another issue... ?

I removed chpl from my system, did a fresh download then

```
./configure
make
buddha314@notes:~/chapel$ make install
Makefile:19: ../make/Makefile.base: No such file or directory
make: *** No rule to make target '../make/Makefile.base'.  Stop.
buddha314@notes:~/chapel$
```

Removed it, reinstall.

```
buddha314@notes:~/chapel$ git clone https://github.com/chapel-lang/chapel.git
buddha314@notes:~/chapel$ cd chapel
buddha314@notes:~/chapel$ source util/setchplenv.bash
buddha314@notes:~/chapel$ make
buddha314@notes:~/chapel$ make install
...
***** src/ *****
***** ./ *****
cd modules && make
make[2]: Nothing to be done for 'default'.
Exiting: Installation path not configured.
         See ./configure --help
Makefile:190: recipe for target 'install' failed
make: *** [install] Error 255
```
so I did

```
./configure
buddha314@notes:~/chapel$ make install
...cp: cannot create regular file '/usr/local/lib/chapel/1.16/third-party/re2/Makefile': Permission denied
cp: cannot create regular file '/usr/local/lib/chapel/1.16/third-party/re2/Makefile.include': Permission denied
cp: cannot create regular file '/usr/local/lib/chapel/1.16/third-party/utf8-decoder/utf8-decoder.h': Permission denied
cp: cannot create regular file '/usr/local/lib/chapel/1.16/chplconfig': Permission denied
find: ‘/usr/local/share/chapel/1.16’: No such file or directory
```

so 

```
buddha314@notes:~/chapel$ sudo make install
Makefile:19: ../make/Makefile.base: No such file or directory
make: *** No rule to make target '../make/Makefile.base'.  Stop.
```

--- a/util/buildRelease/install.sh
+++ b/util/buildRelease/install.sh
@@ -152,26 +152,78 @@ myinstalldir () {
   FROM="$CHPL_HOME/$1"
   TO="$2"
   #echo myinstalldir $FROM $TO
-  mkdir -p $TO
-  ( cd $FROM ; tar cf - . ) | ( cd $TO ; tar xf - )
-  return $?
+  mkdir -p "$TO"
+
+  if [ ! -d "$FROM" ]
+  then
+    echo "Error: source directory '$FROM' missing"
+    exit -1
+  fi
+  if [ ! -d "$TO" ]
+  then
+    echo "Error: could not find/create destination directory '$TO'"
+    exit -1
+  fi
+
+  ( cd "$FROM" ; tar cf - . ) | ( cd "$TO" ; tar xf - )
+
+  if [ $? -ne 0 ]
+  then
+    echo "Error: failed directory copy '$FROM' '$TO'"
+    exit -1
+  fi
 }
 
 myinstallfile () {
   FROM="$CHPL_HOME/$1"
   TO="$2"
-  mkdir -p $TO
+
+  mkdir -p "$TO"
+
+  if [ ! -f "$FROM" ]
+  then
+    echo "Error: source file '$FROM' missing"
+    exit -1
+  fi
+  if [ ! -d "$TO" ]
+  then
+    echo "Error: could not find/create destination directory '$TO'"
+    exit -1
+  fi
+
   #echo myinstallfile $FROM $TO
-  cp $FROM $TO
-  return $?
+  cp "$FROM" "$TO"
+
+  if [ $? -ne 0 ]
+  then
+    echo "Error: failed cp '$FROM' '$TO'"
+    exit -1
+  fi
 }
 
 myinstallfileto () {
   FROM="$CHPL_HOME/$1"
   TO="$2"
   #echo myinstallfile $FROM $TO
-  cp $FROM $TO
-  return $?
+  if [ ! -f "$FROM" ]
+  then
+    echo "Error: source file '$FROM' missing"
+    exit -1
+  fi
+
+  cp "$FROM" "$TO"
+
+  if [ $? -ne 0 ]
+  then
+    echo "Error: failed cp '$FROM' '$TO'"
+    exit -1
+  fi
+
+  if [ ! -f "$TO" ]
+  then
+    echo "Error: could not install to '$TO'"
+    exit -1
+  fi
 }
 
 
@@ -206,7 +258,7 @@ myinstallfile README.rst              "$DEST_CHPL_HOME"
 # copy modules
 myinstalldir  modules                 "$DEST_CHPL_HOME"/modules
 
-# copy util/printchplenv 
+# copy util/printchplenv
 myinstallfile util/printchplenv       "$DEST_CHPL_HOME"/util/
 
 # copy util/chplenv

